wpilibj/src/athena/cpp/lib/HALUtil.cpp

```diff
@@ -52,7 +56,7 @@ static jclass pwmConfigDataResultCls = nullptr;
 
 namespace frc {
 
-static void GetStackTrace(JNIEnv *env, std::string &res, std::string &func) {
+static void GetStackTrace(JNIEnv *env, llvm::SmallString<128> &res, llvm::SmallString<128> &func) {
```

Comment:
Doing this doesn't provide the operators from StringRef however. Is there a way to add that?